forms/HtmlEditorField.php

```diff
@@ -323,8 +323,12 @@ public function MediaForm() {
 		$fileField->setAttribute('data-multiselect', true);
 		$columns = $fileField->getConfig()->getComponentByType('GridFieldDataColumns');
 		$columns->setDisplayFields(array(
-			'CMSThumbnail' => false,
-			'Name' => _t('File.Name'),
+			'StripThumbnail' => false,
+			'Title' => _t('File.Title'),
+			'Created' => singleton('File')->fieldLabel('Created'),
+		));
+		$columns->setFieldCasting(array(
+			'Created' => 'Date->Nice'
```

Comment:
Cool just go with it - we can just keep the Uploaded/Created/Added label thing in mind, looking good